assets/sass/_fonts.scss

```diff
@@ -1,2 +1,3 @@
 @import url('https://fonts.googleapis.com/css?family=Open+Sans:400,400i,400,700|Ubuntu:400,700');
 @import url('https://fonts.googleapis.com/css?family=PT+Mono');
+@import url('https://fonts.googleapis.com/css?family=Material+Icons');
```

Comment:
I wanted to use the same material "copy" button we use on https://app.pulumi.com. As a follow-up, would anyone be opposed to removing our use of FontAwesome and just using the Material icons instead?

Reply:
Personally I’m open to having a look at how using Material icons affects the site overall, if that means being able to settle on a single icon set — but more generally, I’m honestly not in favor of propagating concrete Material design elements (as opposed to just guidance, which is fine) outside of the console.